core/src/main/java/org/keycloak/Config.java

```diff
@@ -235,6 +235,13 @@ public static interface Scope {
 
         Scope scope(String... scope);
 
+        /**
+         * @deprecated since 26.2.3, to be removed
```

Comment:
```suggestion
         * @deprecated since 26.2.6, to be removed
```

nit: I believe it'll fit into the 26.2.6